File: Anyelo Carrillo python/Ley Ohm.py
Voltaje = int(input('Ingrese el valor del voltaje del circuito: '))
Resistencia = int(input('Ingrese el valor de la resistencia del circuito: '))
Intensidad = Voltaje/Resistencia #Intesidad entiendase por amperaje
print('Al conectar un resistor de R',Resistencia,'ohm'' a una fuente de V',Voltaje,'voltage circulara una corriente de',Intensidad,'amperios')

File: Anyelo Carrillo Go/Ley Ohm.go
package main

import (
	"fmt"
)

func main() {
	// Solicitar al usuario que ingrese el valor del voltaje
	fmt.Print("Ingrese el valor del voltaje del circuito: ")
	var Voltaje int
	fmt.Scanln(&Voltaje)

	// Solicitar al usuario que ingrese el valor de la resistencia
	fmt.Print("Ingrese el valor de la resistencia del circuito: ")
	var Resistencia int
	fmt.Scanln(&Resistencia)

	// Calcular la intensidad (amperaje)
	Intensidad := float64(Voltaje) / float64(Resistencia)

	// Mostrar el resultado
	fmt.Printf("Al conectar un resistor de R %d ohm a una fuente de V %d voltios, circulará una corriente de %.2f amperios\n", Resistencia, Voltaje, Intensidad)
}
